lib/bloc_works/controller.rb

```diff
@@ -0,0 +1,35 @@
+require 'erubis'
+
+module BlocWorks
+	class Controller
+		def initialize(env)
+			@env = env
+		end
+
+		def render(view, locals = {})
+			filename = File.join("app", "views", controller_dir, "#{view}.html.erb")
+			template = File.read(filename)
+			eruby = Erubis::Eruby.new(template)
+			# get the instance variables of the controller with #instance_variables
+			# this returns an array of instance_variables
+			# iterate, use #instance_variable_get to get values & add to {locals}
+			self.instance_variables.each do |var|
+				locals[var] = self.instance_variable_get(var)
+			end
+			eruby.result(locals.merge(env: @env))
```

Comment:
`env` is also an instance variable (see constructor). This does not break anything, but it is redundant.